### DISCUSSION TITLE
Chains - List the chains your project supports (Chia is not an option)

### DISCUSSION THREAD

**Discussion by darkverbito** [WalletConnect/walletconnect-monorepo#2604] [Developer Support]
I want to register my wallet in the explorer. However, my wallet only supports Chia (XCH) but this is not an option and something must be selected. Can you please add Chia as a Chain option?

**Comment by glitch-txs** [comment]
Hi you can ask for Chia support, you can follow the docs https://docs.walletconnect.com/2.0/advanced/multichain/overview#register-chain-with-the-explorer
